fabfile/run.py

```diff
@@ -198,3 +199,42 @@ def tpch_queries(query_info, connectionURI, pg_version, citus_version):
         out_val = run(run_string)
         results_file.write(out_val)
         results_file.write('\n')
+
+@task
+@runs_once
+@roles('master')
+def update():
+     citus_versions = ['release-8.0', 'release-8.1']
+     pg_version = '11.5'
```

Comment:
similarly, instead of a hardcoded value, can this be a parameter as well?

Reply:
An alternative solution is to use `config.settings['pg-version']`
This way we can override this by adding some initial use commands. For example

	fab use.postgres:12.0 run.update:8.1:8.2